go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/gin-contrib/gzip v0.0.1
 	github.com/gin-gonic/gin v1.4.0
 	github.com/go-sql-driver/mysql v1.4.1
+	github.com/gorilla/websocket v1.4.1 // indirect
```

Comment:
This is used directly in vmaas_sync.go so remove `// indirect`, please. Btw., use `go mod tidy` command to clean up deps.

Reply:
Fixed